djangoproject/templates/contact/foundation.html

```diff
@@ -32,18 +32,22 @@ <h1>{% translate "Contact the Django Software Foundation" %}</h1>
   <form action="." method="post" accept-charset="utf-8" class="form-input">
     {% csrf_token %}
     <p>
+      <label for="{{ form.name.id_for_label }}">{{ form.name.label }}</label>
```

Comment:
You could use:
```suggestion
      {{ form.name.label_tag }}
```

It's not strictly equivalent since it will add a `:` after the label but I think that's not a bad change.

Reply:
Oh I like this!